src/test/java/com/dell/isg/smi/service/server/configuration/manager/ConfigurationManagerImplTest.java

```diff
@@ -0,0 +1,219 @@
+package com.dell.isg.smi.service.server.configuration.manager;
+
+import com.dell.isg.smi.service.server.configuration.model.*;
+import org.eclipse.persistence.jaxb.JAXBContextFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.bind.Binder;
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Marshaller;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.junit.Assert.*;
+
+public class ConfigurationManagerImplTest {
+    private IConfigurationManager configManager;
+
+    @Before
+    public void setup() {
+        ConfigurationManagerImpl impl = new ConfigurationManagerImpl();
+        // TODO: figure out how to get this auto-wired
+        impl.setComponentPredicate(new ComponentPredicate());
+        configManager = impl;
```

Comment:
This isn't needed any more since `updateConfiguration` doesn't use `componentPredicate` anymore. So `setup()` and the private variable can be removed and just do `new ConfigurationManagerImpl()`